source/parameters.rst

```diff
@@ -0,0 +1,204 @@
+        active = boolean(default=True)
+        # The grid to use
+        grid = string(default="")
+        qos = string(default="regular")
+        # The number of nodes to use
+        nodes = integer(default=1)
+        # The maximum time to run
+        walltime = string(default="02:00:00")
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        reference_data_path = string(default="")
+        # Used for `test_name` and `short_test_name` in https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        short_name = string(default="")
+        cfg = string(default="")
+        # The sets to run
+        # Note that "enso_diags","qbo","area_mean_time_series" require time-series data.
+        # They also require `obs_ts` and `ref_start_yr` to be set.
+        # "qbo" requires `ref_final_yr` to be set as well.
+        # "diurnal_cycle" requires `climo_diurnal_subsection` and `ds_obs_climo` to be set.
+        sets = string_list(default=list("lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","area_mean_time_series","diurnal_cycle"))
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        backend = string(default="mpl")
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        diff_title = string(default="Model - Observations")
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        run_type = string(default="model_vs_obs")
+        # Used to label the results directory
+        tag = string(default="model_vs_obs")
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        output_format = string_list(default=list("png"))
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        output_format_subplot = string_list(default=list())
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        multiprocessing = boolean(default=True)
+        # See https://e3sm-project.github.io/e3sm_diags/_build/html/master/available-parameters.html
+        num_workers = integer(default=24)
+        # Variables to process
+        vars = string(default="FSNTOA,FLUT,FSNT,FLNT,FSNS,FLNS,SHFLX,QFLX,TAUX,TAUY,PRECC,PRECL,PRECSC,PRECSL,TS,TREFHT,CLDTOT,CLDHGH,CLDMED,CLDLOW,U")
+        # Name of the subsection of `[climo]` to use for "diurnal_cycle" runs
+        climo_diurnal_subsection = string(default="")
+        # The following two parameters aren't defined in `default.ini`
+        # Path to observation data for diurnal cycle runs
+        dc_obs_climo = string
+        # Path to observation data for time-series-required ("enso_diags","qbo","area_mean_time_series") runs
+        obs_ts = string
```

Comment:
These don't have default values in `default.ini`. I also need to add descriptions for `ref_start_yr`, `ref_end_yr`, and `ref_final_yr`.